module/bdev/ocf/vbdev_ocf.c

```diff
@@ -1619,6 +1624,9 @@ vbdev_ocf_examine_disk(struct spdk_bdev *bdev)
 	const char *bdev_name = spdk_bdev_get_name(bdev);
 	struct vbdev_ocf *vbdev;
 
+	if (getenv("ocf_force"))
```

Comment:
Is this needed? Is it different from --force flag?

Reply:
fixed